app/backend/aquifers/change_history.py

```diff
@@ -0,0 +1,185 @@
+"""
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import json
+
+from django.contrib.gis.geos import GEOSGeometry
+
+from aquifers.models import Aquifer, VerticalAquiferExtentsHistory
+
+FIELDS_TO_IGNORE = (
+    'create_user',
+    'create_date',
+    'update_user',
+    'update_date',
+    'geom_simplified',
+    'well',
+    'activitysubmission',
+    'resources',
+    'verticalaquiferextent',
+    'update_to_aquifer_set',
+    'update_from_aquifer_set',
+    'history',
+)
+KEY_VALUE_LOOKUP = {
+    'material': 'description',
+    'subtype': 'description',
+    'vulnerability': 'description',
+    'productivity': 'description',
+    'demand': 'description',
+    'known_water_use': 'description',
+    'quality_concern': 'description',
+}
+
+def get_aquifer_history(aquifer):
+    history = []
+    history += get_aquifer_reversion_history(aquifer)
+    history += get_vertical_aquifer_extents_history(aquifer)
+
+    history.sort(key=get_history_date, reverse=True)
+
+    return history
+
+def get_aquifer_reversion_history(aquifer):
+    # pylint: disable=protected-access
```

Comment:
I really like the docstrings in gwells` change_history.py describing what the function does and what it returns. Can we add them here too?

I also noticed that it shares a similar codebase, generating a history diff. Is there a reason to name them differently (history vs history_diff) and does it make more sense to drop the "diff" in this context?

Reply:
Yea I can make that change.